cassiopeia/preprocess/pipeline.py

```diff
@@ -464,70 +464,71 @@ def align_sequences(
     ref: Optional[str] = None,
     gap_open_penalty: float = 20,
     gap_extend_penalty: float = 1,
+    method: Literal["local", "global"] = "local",
     n_threads: int = 1,
 ) -> pd.DataFrame:
     """Align reads to the TargetSite reference.
 
     Take in several queries stored in a DataFrame mapping cellBC-UMIs to a
-    sequence of interest and align each to a reference sequence. The alignment
-    algorithm used is the Smith-Waterman local alignment algorithm. The desired
-    output consists of the best alignment score and the CIGAR string storing the
-    indel locations in the query sequence.
+    sequence of interest and align each to a reference sequence. Either local
+    or global alignment may be performed, depending on the `method` argument.
+    The defaults for the gap open and gap extend penalties were selected via
+    in-silico simulation (which also happens to be equivalent to the values
+    used in the GESTALT technology described in McKenna et. al, 2016).
```

Comment:
I'm not sure we should say this here -- because they differ in absolute terms.

Reply:
How about now?